aptos-node/src/bootstrap.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, Result};
+use aptos_config::config::NodeConfig;
+use aptos_db::AptosDB;
+use aptos_storage_interface::DbReaderWriter;
+use std::sync::Arc;
+use tokio::runtime::Runtime;
+
+#[cfg(not(feature = "consensus-only-perf-test"))]
+pub(crate) fn bootstrap_db(
+    node_config: &NodeConfig,
+) -> Result<(Arc<AptosDB>, DbReaderWriter, Option<Runtime>)> {
+    use aptos_backup_service::start_backup_service;
+
+    let (aptos_db, db_rw) = DbReaderWriter::wrap(
+        AptosDB::open(
```

Comment:
Could this remove the repeated open?
https://gist.github.com/bchocho/7a8d5c17c718e7322cab95d6fa7d3189

Reply:
Thanks Brian, it works. I incorporated it.